code/api/src/modules/user/resolvers.js

```diff
@@ -27,6 +27,26 @@ export async function create(parentValue, { name, email, password }) {
   }
 }
 
+// Update user
+export async function update(parentValue, { id, image }, { auth }) {
+  if(auth.user) {
+    const user = await models.User.update(
+      {
+        image
+      },
+      {where: {id}}
+    )
+    
+    if(!user) {
+      throw Error('No user exists.')
+    } else {
+      return await models.User.findOne({ where: { id } })
+    };
+  } else {
+    throw new Error('Operation denied.')
+  }
+}
+
```

Comment:
I actually think I understand what is going on here! So when a user wants to upload/update an image, you are using the id as an identifier for the user so the image knows where to go. The syntax is obviously a little different, but it looks good! Also, nice error handling.